drivers/hmis/lib/form_data/records/client.json

```diff
@@ -2,129 +2,131 @@
   "item": [
     {
       "type": "GROUP",
-      "link_id": "first-name-group",
+      "link_id": "name",
       "text": "Name",
       "required": false,
       "item": [
         {
-          "type": "STRING",
-          "required": false,
-          "link_id": "first-name",
-          "text": "First Name",
-          "field_name": "firstName"
-        },
-        {
-          "type": "STRING",
-          "required": false,
-          "link_id": "preferred-name",
-          "text": "Preferred Name",
-          "field_name": "preferredName"
-        },
-        {
-          "type": "STRING",
-          "required": false,
-          "link_id": "middle-name",
-          "text": "Middle Name",
-          "field_name": "middleName"
-        },
-        {
-          "type": "STRING",
-          "required": false,
-          "link_id": "last-name",
-          "text": "Last Name",
-          "field_name": "lastName"
-        },
-        {
-          "type": "STRING",
-          "required": false,
-          "link_id": "name-suffix",
-          "text": "Suffix",
-          "field_name": "nameSuffix"
-        },
-        {
-          "type": "CHOICE",
-          "required": false,
-          "link_id": "name-dq",
-          "text": "Name Data Quality",
-          "field_name": "nameDataQuality",
-          "pick_list_reference": "NameDataQuality"
-        },
-        {
-          "type": "OPEN_CHOICE",
-          "required": false,
-          "link_id": "pronouns",
-          "text": "Pronouns",
-          "repeats": true,
-          "field_name": "pronouns",
-          "pick_list_options": [
+          "type": "GROUP",
+          "link_id": "fn-group",
+          "component": "HORIZONTAL_GROUP",
+          "item": [
             {
-              "code": "she/her"
+              "type": "STRING",
+              "required": false,
+              "link_id": "first-name",
+              "text": "First Name",
+              "field_name": "firstName"
             },
             {
-              "code": "he/him"
+              "type": "STRING",
+              "required": false,
+              "link_id": "preferred-name",
+              "text": "Preferred Name",
+              "field_name": "preferredName"
             },
```

Comment:
I don't know why, but I wonder if we should collect preferred name either first, or after name.  It seems odd to type Jane "Trish" Apricot Smith.  Seems more normal to type:
Trish
Jane Apricot Smith

Reply:
Ah yeah, that's a good point. I'm reviewing these with Ash tomorrow, I'll see what she thinks as well.